Makefile

```diff
@@ -110,7 +110,7 @@ lint: ## check style with flake8 and isort
 
 .PHONY: fix-lint
 fix-lint: ## fix lint issues using autoflake, autopep8, and isort
-	find sdv tests -name '*.py' | xargs autoflake --in-place --remove-all-unused-imports --remove-unused-variables
+	find sdv tests -name '*.py' | xargs autoflake --in-place --remove-all-unused-imports --remove-unused-variables --ignore-init-module-imports
```

Comment:
I think we should remove `--ignore-init-module-imports`, this affects to all `__init__.py` files and I think we would like to keep this check.